README.md

```diff
@@ -73,15 +76,25 @@ This will create an install directory with the installed libraries as well as ot
 
 ## Setup
 
-The environment must be set by activating the setup script (`install/bin/setup_histfitter.sh`) in the directory where you want to work. This will create five new directories in your work directory: `analysis`, `config`, `data`, `macros`, and `results`. It also copies the `HistFactorySchema.dtd` file into the config directory. Most importantly, the setup script sets the environment variable paths that are required for the scripts to work.  The setup also adds the analysis directory in the working area to the Python path.  This makes it convenient to import additional python files without needing to install them in the build area.
+The environment must be set by activating the setup script (`install/bin/setup_histfitter.sh`) in the directory where you want to work. This will create three new directories in your work directory: `config`, `data`, and `results`. It also copies the `HistFactorySchema.dtd` file into the config directory. Most importantly, the setup script sets the environment variable paths that are required for the scripts to work. Use the `-e` flag to obtain a copy of the `analysis` and `macros` folders containing the examples, which are a good starting point for creating your own analysis. If you want to import additional python files, make sure that they are located in a folder which is added to the `PYTHONPATH` environment variable (like `analysis`).
```

Comment:
Ah yes, I made this PR first, then realised while updating the tutorial that it is just really confusing. My initial PR (1.3.0) copied /analysis, but it was changed later so that's why the documentation and functionality was different. I will update everything to fit with the new solution (make /analysis folder, but not copy /tutorial unless -e flag is set) since you agreed with that :)